.circleci/config.yml

```diff
@@ -64,6 +79,128 @@ jobs:
+            - run: npm install
+            - rust/format:
+                with_cache: false # See CACHE_FALSE below for why
+            - rust/clippy:
+                with_cache: false # See CACHE_FALSE below for why
+            - rust/test:
+                with_cache: false # See CACHE_FALSE below for why
+      - when:
+          condition:
+            equal: [ *rust_windows_executor, << parameters.platform >> ]
+          steps:
+            - run:
+                name: Install rustup
+                command: |
+                  $installer_dir = "$Env:TEMP"
+                  echo "Downloading rustup"
+                  (New-Object System.Net.WebClient).DownloadFile("https://win.rustup.rs", "$installer_dir\rustup-init.exe")
+                  echo "Installing rustup"
+                  & $installer_dir\rustup-init.exe --profile minimal --component rustfmt,clippy -y
+                  echo "Done"
+            - run:
+                name: Special case for Windows because of ssh-agent
+                command: |
+                  Add-Content -path "${Env:USERPROFILE}\.cargo\config.toml" @"
+                  [net]
+                  git-fetch-with-cli = true
+                  "@
+            - run:
+                name: Install nvm (with choco)
+                command: choco install nvm
+            - run:
+                name: Install desired Node.js version with nvm
+                command: |
+                  nvm install ${Env:NODE_VERSION}
+                  nvm on
+            - run:
+                name: Debug output of Node.js and npm versions
+                command: |
+                  echo 'Node.js version:'
+                  node --version
+                  echo 'npm version:'
+                  npm --version
+            - run: npm install
+            - rust/format:
+                with_cache: false # See CACHE_FALSE below for why
+            - rust/clippy:
+                with_cache: false # See CACHE_FALSE below for why
+            - rust/test:
+                with_cache: false # See CACHE_FALSE below for why
+
+# CACHE_FALSE: A special case
+#
+# The caches above on the Rust orb jobs are all disabled intentionally because
+# the `cache_version` they are offering in their keys is, I think, misplaced.
+# The misplacement is resulting in caches being restored to hosts of differing
+# architectures than the host that created the cache. While this might work for
+# some common-foundation architectures (e.g. Unix vs Linux, with the same
+# usernames), it breaks down with even the slightest variation (e.g., macOS has
+# different home directory) structures, so the cache is attempted to be
+# restored to $HOME paths with different user, which fails.
+#
+# I've opened https://github.com/CircleCI-Public/rust-orb/issues/10 to capture
+# this bug, and I've opened https://github.com/CircleCI-Public/rust-orb/pull/11
+# to hopefully fix it.  Once this lands, each `with_cache: false` line above
+# can be removed.
```

Comment:
```suggestion
# to hopefully fix it.  Once this lands, each `with_cache: false` line above
# can be changed:
#   - Only the first job in the list needs caching enabled
#     (otherwise, we'll pay the cost of restore for already-restored content)
#   - They should have 'cache_version' parameters passed with the platform:
#     e.g. 'cache_version: v1-mac'
```